app/javascript/packs/base.jsx

```diff
@@ -4,6 +4,40 @@ import {
   getInstantClick,
   initializeTouchDevice,
 } from '../topNavigation/utilities';
+import { fetchSearch } from '../utilities/search';
```

Comment:
Refactor (blocking): Use the webpack alias `@utilities` instead.

```suggestion
import { fetchSearch } from '@utilities/search';
```